src/ui/screens/messaging/ViewMessagesInbox.qml

```diff
@@ -87,15 +87,12 @@ VPNViewBase {
             _searchBarHasError: !root.isEmptyState && listView.count === 0
 
             _filterProxySource: VPNAddonManager
-            _filterProxyCallback: obj => {
+            _filterProxyCallback: obj => obj.addon.type === "message" && obj.addon.containsSearchString(getSearchBarText())
+            _sortProxyCallback: (obj1, obj2) => obj1.addon.date > obj2.addon.date
+            _editCallback: () => {
                                       root.closeAllSwipes()
```

Comment:
If the model is refreshing with every new search query, do we even need to call `closeAllSwipes` since they close by default when the cells are re-initialized?

Reply:
You are right!